DocsSearchProvider.cs

```diff
@@ -24,29 +25,32 @@ public static class DocsSearchProvider
     /// </summary>
     static string packageFolderName = $"Packages/{packageName}";
 
-    [SearchItemProvider]
+    [UsedImplicitly, SearchItemProvider]
     public static SearchProvider CreateProvider()
     {
         return new SearchProvider("ch.sttz.quicksearch-docs", "Docs") {
             filterId = "docs:",
-            fetchItems = (context, items, provider) => {
-                var tokens = context.tokenizedSearchQueryLower;
-                var results = GetSearchResults(tokens, context.searchQuery);
-                foreach (var result in results) {
-                    items.Add(provider.CreateItem(
-                        "ch.sttz.quicksearch-docs." + result.url,
-                        -result.score,
-                        result.title,
-                        result.description,
-                        Icons[result.type],
-                        result
-                    ));
-                }
-            }
```

Comment:
@sttz in case you would not want at all the async version of `fetchItems` you could simply return null at the end of the lambda assigned to `fetchItems` and continue to fill the `items` list. Filling `items` works as before were items are piped synchronously.